memflow/src/architecture/mod.rs

```diff
@@ -84,7 +85,7 @@ pub trait ScopedVirtualTranslate: Clone + Copy + Send {
         addrs: VI,
         out: &mut VO,
         out_fail: &mut FO,
-        arena: &Bump,
+        tmp_buf: &mut [std::mem::MaybeUninit<u8>],
```

Comment:
Is this safe?

Reply:
MaybeUninit is safe to pass around, only creating it is not safe.